docs/root/intro/arch_overview/upstream/load_balancing/load_balancers.rst

```diff
@@ -132,16 +132,18 @@ host B has a weight of 2, then host A will have 21,846 entries and host B will h
 regardless of the configured host and locality weights, so in some extreme cases the actual
 proportions may differ from the configured weights. For example, if the total number of hosts is
 larger than the fixed table size, then some hosts will get 1 entry each and the rest will get 0,
-regardless of weight. Best practice is to monitor the :ref:`min_entries_per_host and
-max_entries_per_host gauges <config_cluster_manager_cluster_stats_maglev_lb>` to ensure no hosts
-are underrepresented or missing.
+regardless of weight.
 
 In general, when compared to the ring hash ("ketama") algorithm, Maglev has substantially faster
 table lookup build times as well as host selection times (approximately 10x and 5x respectively
 when using a large ring size of 256K entries). The downside of Maglev is that it is not as stable
 as ring hash. More keys will move position when hosts are removed (simulations show approximately
-double the keys will move). With that said, for many applications including Redis, Maglev is very
-likely a superior drop in replacement for ring hash. The advanced reader can use
+double the keys will move). Maglev aims for minimal disruption when set of upstream hosts change.
```

Comment:
Main point to explain this with table_size config.  This is what https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto#config-cluster-v3-cluster-maglevlbconfig says.
Does this contradict with previous statement 
"The downside of Maglev is that it is not as stable
as ring hash. More keys will move position when hosts are removed (simulations show approximately
double the keys will move)" ?

Is there a better way to clarify it?

Reply:
@mattklein123 any thoughts?

Reply:
I'm a little confused what we are trying to clarify here. From a very quick glance this is mostly just moving text around?

/wait-any

Reply:
"Maglev aims for minimal disruption when set of upstream hosts change."  this part of the text is copied from Maglev config. Also linked the table_size from here. So you are right - this part is moving the text around.

My question is we have this statement 
" The downside of Maglev is that it is not as stable
as ring hash. More keys will move position when hosts are removed (simulations show approximately
double the keys will move)" in this section 

and 
""Maglev aims for minimal disruption when set of upstream hosts change."  is there is in MaglevLbConfig section.

So when folks reading this section think that MagLev does not handle upstream hosts change well. But the second statement seem to contradict that.

The goal of this PR is to clarify that part and add table_size reference here.

Reply:
I think the original idea behind this text is to say something like "Maglev aims for minimal disruption when a set of upstream hosts change, but the disruption is generally greater than that achieved by Ketama/Ring Hash." I think the text does basically say this though, so feel free to clarify in a way that makes more sense to you?

/wait

Reply:
Sure. Will change. Another thing I got confused with this is this https://docs.google.com/document/d/1IU4b76AgOXijNa4sew1gfBfSiOMbZNiEt5Dhis8QpYg/edit?disco=AAAAMzmbMm8.

So basically it looks like the MagLev disruption is more compared to Ring Hash.  I will reword as you suggested

Reply:
@mattklein123 updated. PTAL if this change makes sense.